src/index.js

```diff
@@ -199,6 +199,14 @@ export default class UkModulusChecking {
    */
 
   isValid() {
+    if (this.sortCode.length !== 6) {
+      return false;
+    }
+
+    if (this.accountNumber.length < 6 || this.accountNumber.length > 10) {
```

Comment:
What about `if(!_.inRange(this.accountNumber, 6, 11) || this.sortCode.length !== 6)`?